test/TestUtils.py

```diff
@@ -258,3 +259,33 @@ def mockreturn(options):
     monkeypatch.setattr(utils, 'get_yaml_files', mockreturn)
     result = utils.get_playbooks_and_roles(options)
     assert result == ['bar/playbook.yml']
+
+
+_DEFAULT_RULEDIRS = [constants.DEFAULT_RULESDIR]
+_CUSTOM_RULESDIR = Path(__file__).parent / "custom_rules"
+_CUSTOM_RULEDIRS = [
+    str(_CUSTOM_RULESDIR / "example_inc"),
+    str(_CUSTOM_RULESDIR / "example_com")
+]
+
+
+@pytest.mark.parametrize(("uruledirs", "use_default", "expected"), (
+    pytest.param([], True, _DEFAULT_RULEDIRS),
+    pytest.param([], False, _DEFAULT_RULEDIRS),
+    pytest.param(_CUSTOM_RULEDIRS, True, _CUSTOM_RULEDIRS + _DEFAULT_RULEDIRS),
+    pytest.param(_CUSTOM_RULEDIRS, False, _CUSTOM_RULEDIRS)
+))
+def test_get_rules_dirs(uruledirs, use_default, expected):
+    assert utils.get_rules_dirs(uruledirs, use_default) == expected
+
+
+@pytest.mark.parametrize(("uruledirs", "use_default", "expected"), (
+    pytest.param([], True, sorted(_CUSTOM_RULEDIRS) + _DEFAULT_RULEDIRS),
+    pytest.param([], False, sorted(_CUSTOM_RULEDIRS) + _DEFAULT_RULEDIRS),
+    pytest.param(_CUSTOM_RULEDIRS, True,
+                 _CUSTOM_RULEDIRS + sorted(_CUSTOM_RULEDIRS) + _DEFAULT_RULEDIRS),
+    pytest.param(_CUSTOM_RULEDIRS, False, _CUSTOM_RULEDIRS)
+))
+def test_get_rules_dirs_with_custom_rules(uruledirs, use_default, expected, monkeypatch):
```

Comment:
`uruledirs` is a weird name. What does it mean? Could you make it more explicit, please?

Reply:
Also thanks for pointing it out! Renamed it in the latest commit.